src/EnergyPlus/DesiccantDehumidifiers.cc

```diff
@@ -123,6 +123,7 @@ namespace DesiccantDehumidifiers {
     //              (Please see copyright and disclaimer information at end of module)
 
     static std::string const fluidNameSteam("STEAM");
+    Real64 constexpr TempSteamIn = 100.0;
```

Comment:
DataGlobalConstants.hh:

    Real64 constexpr SteamInitConvTemp = 100.0;             // [deg C], standard init steam vol to mass flow conversion temp